internal/generate/collector/collect.go

```diff
@@ -47,6 +47,15 @@ type Manifests struct {
 	Others                           []unstructured.Unstructured
 }
 
+var (
+	roleGK                 = rbacv1.SchemeGroupVersion.WithKind("Role").GroupKind()
+	clusterRoleGK          = rbacv1.SchemeGroupVersion.WithKind("ClusterRole").GroupKind()
+	deploymentGK           = appsv1.SchemeGroupVersion.WithKind("Deployment").GroupKind()
+	crdGK                  = apiextv1.SchemeGroupVersion.WithKind("CustomResourceDefinition").GroupKind()
```

Comment:
We need to continue supporting v1beta1 CRDs for now.
```suggestion
	v1crdGK                  = apiextv1.SchemeGroupVersion.WithKind("CustomResourceDefinition").GroupKind()
	v1beta1crdGK                  = apiextv1beta1.SchemeGroupVersion.WithKind("CustomResourceDefinition").GroupKind()
```